CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.9.0
+
+* New project generator updates
+  * bump NervesPack to 0.4 which drops `:nerves_firwmare_ssh` in
```

Comment:
```suggestion
  * Bump [NervesPack to 0.4](https://hexdocs.pm/nerves_pack/changelog.html#v0-4-0) which drops `:nerves_firwmare_ssh` in
```

Adding a link to `nerves_pack`, since that seems helpful for such a big change.